server/tests/util/db.js

```diff
@@ -1,46 +1,34 @@
 const { dbClient, schema } = require('../../db');
 
 async function seedDatabase() {
-  // Create collections if needed
-  await Promise.all(schema.map(async (schemaItem) => {
-    await dbClient.db.createCollection(schemaItem.collection);
-  }));
-
-  // Create collection indexes if needed
-  const results = [];
-  schema.forEach((schemaItem) => {
-    const schemaItemCollection = dbClient.db.collection(schemaItem.collection);
-    schemaItem.indexes.forEach((index) => {
-      results.push(schemaItemCollection.createIndex({ [index.key]: 1 }, index.options));
-    });
-  });
-
-  await Promise.all(results); // Wait for all synchronous operations to pass/fail
+  /* eslint-disable no-await-in-loop */
+  /* eslint-disable no-restricted-syntax */
```

Comment:
We should consider avoiding these warnings. At the very least, we should re-enable the linting rules after this little loop.

Reply:
True. There are some implications using **async** in a **forEach** instead of a **for** in this code, but since it's just a seed script I think we are fine just enabling the lint rules for now.